src/fakeData/fakeFacilities.ts

```diff
@@ -0,0 +1,43 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
+import { faker, fakerJA } from '@faker-js/faker'
+
+export function generateRandomCreateFacilityInput(healthcareProfessionalIds?: string[])
+    : gqlTypes.CreateFacilityInput {
+    const sex = faker.person.sexType()
+    const firstName = faker.person.firstName()
+    const lastName = faker.person.lastName()
+    const fullEnglishName = faker.person.fullName({ firstName, lastName, sex })
```

Comment:
Is there a reason the `sex` is being included in the name? This seems like something that should be a completely separate field.

Reply:
It how the fake data generator can use a consistent gendered name across fields.